Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM python:3.8-slim
 
 ENV PIP_NO_CACHE_DIR=1 \
     JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64/ \
+    PYTHONPATH=/app \
```

Comment:
Ahhhh there was the missing part.

Reply:
Yea this was required otherwise the imports would all fail. I thought that python included everything in the current working directory as part of the `PYTHONPATH` but I guess not 🤷‍♀️